practices/practice-one/src/styles/common/index.css

```diff
@@ -0,0 +1,10 @@
+.app {
```

Comment:
Why app in common?

Reply:
-  I will add it to components/default.css

Reply:
Please help to create a ticket for it, then keep in here

Reply:
I will fix this via #32